extensions/amp-ad-network-doubleclick-impl/amp-ad-network-doubleclick-impl-internal.md

```diff
@@ -142,6 +142,7 @@ Supported via `json` attribute:
     -   `"<key_string>":["<value1>", "<value2>", ...]`. See below for example.
 -   `ppid`: Sets a custom provided user ID for targeting. Do not set when
     serving responses to crawlers since this value is expected to be dynamic.
+-   `macr`: Specify maximum content rating for ad in compliance with UK's Age-Appropriate Design Code (AADC).
```

Comment:
I don't think we should mention the AADC -- while the regulation is a motivation in creating this feature, it is general purpose and not tied to the regulation.  See our existing guidance to publishers: https://www.blog.google/products/admob/use-ad-content-filtering-to-help-p9rh/ https://support.google.com/admob/answer/10477886?hl=en

How about: "specify the maximum content rating of the ads used to fill this slot."?